packages/math-input/src/components/keypad/__tests__/KeypadButton.test.tsx

```diff
@@ -0,0 +1,39 @@
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import * as React from "react";
+import "@testing-library/jest-dom";
+
+import Keys from "../../../data/key-configs";
+import {KeypadButton} from "../keypad-page-items";
+
+describe("<KeypadButton />", () => {
+    it("uses the aria label", () => {
+        // Arrange
+        render(
+            <KeypadButton
+                onClickKey={(config) => jest.fn()}
+                keyConfig={Keys.LEFT_PAREN}
+            />,
+        );
+
+        // Assert
+        expect(
+            screen.getByRole("button", {name: "LEFT_PAREN"}),
+        ).toBeInTheDocument();
+    });
```

Comment:
```suggestion
    it("uses the aria label from the key config", () => {
        // Arrange
        render(
            <KeypadButton
                onClickKey={() => {}}
                keyConfig={Keys.LEFT_PAREN}
            />,
        );

        // Assert
        expect(
            screen.getByRole("button", {name: "LEFT_PAREN"}),
        ).toBeInTheDocument();
    });
```

Adding a little more info to the name and make the click handler a noop.

Reply:
just a quick note—in order to make onClickKey a noop it turns out we do still have to pass in the argument, so in the new commit the callback is now (config) => {}